ontology/docs/building_config.md

```diff
@@ -1,5 +1,9 @@
 # Model Instance Configuration (building configuration file)
```

Comment:
this file should not be synched currently as the code is not synched. revert this part?

Reply:
ty - reverted